src/pages/londonliving.tsx

```diff
@@ -15,7 +14,34 @@ import styles from "./londonliving.module.scss"
 
 import HeaderUnderlay from "../components/header-underlay"
 
-import YouTube from "react-youtube"
+type Episode = {
+    title: string
+    audioUrl: string
+    blurb: string
+}
+
+type Season = {
+    title: string
+    episodes: Episode[]
+}
+
+const PodcastEpisode: React.FC<Episode> = ({ title, audioUrl, blurb }) => {
```

Comment:
could you extract this into `src/pages/londonliving/_components/podcast-episode.tsx` and move the current file into `src/pages/londonliving/index.tsx`?